app/src/main/java/com/example/marvelapp/data/service/api/MarvelApi.kt

```diff
@@ -14,5 +13,5 @@ interface MarvelApi {
     fun getCharacter(): Call<MarvelBaseResponse<MarvelResponse<List<CharacterResponse>>>>
 
     @GET("/v1/public/characters/{characterId}")
-    fun getSingleCharacter(@Path("characterId") id: Int): Call<MarvelBaseResponse<MarvelResponse<List<CharacterDetailResponse>>>>
+    fun getSingleCharacter(@Path("characterId") id: Int): Call<MarvelBaseResponse<MarvelResponse<List<CharacterEntity>>>>
```

Comment:
why did you change this? this is not okay. The service should response something like CharacterDetailResponse

Reply:
Thx, now its fixed, and i am reusing the code that i have in CharacterResponse